### DISCUSSION TITLE
How to proceed with simple bugs that are fixed with one line of code?

### DISCUSSION THREAD

**Discussion by lutzso** [NationalSecurityAgency/ghidra#6413] [Q&A]
I discovered a bug which is fixed by adding one line of code. Do I have to become a developer to get this done or do I need to open an issue? What is the best way to proceed? (By the way, I do not have objections in developing. For me, it is a question of time setting up the environment, and getting everything started.)

The bug in question:
- in class: `ghidra.util.bytesearch.SequenceSearchState`
- in method: `public void apply(InputStream in, long maxBytes, ArrayList<Match> match, TaskMonitor monitor)`
- line 263: `* @param monitor - if non-null, check for user cancel, and maintain progress info`
- and line 268: `long progress = monitor.getProgress();`

Problem:
If one passes "null" for the TaskMonitor object, line 268 throws an exception, which is in contrast to the stated behavior in the docstring in line 263. In the same function in line 341, however, there is a test `monitor != null`.

Solution:
Around line 268, there should be a test for a null-object, as in line 341.

Thank you in advance for guidance

Lutz

**Comment by GhidorahRex** [comment]
You can just open an issue for it, no need to do a pull-request unless you really want to.
